dev-libs/ocl-icd/ocl_icd-2.2.12.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="Alternative to vendor specific OpenCL ICD loaders"
+DESCRIPTION="OpenCL implementations are provided as ICD (Installable Client \
+Driver). An OpenCL program can use several ICD thanks to the use of an ICD \
+Loader as provided by this project. This free ICD Loader can load any (free \
+or non free) ICD."
+HOMEPAGE="http://www.blosc.org/"
+COPYRIGHT="2012-2015 Brice Videau
+	2012-2015 Vincent Danjean"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://forge.imag.fr/frs/download.php/836/ocl-icd-2.2.12.tar.gz"
```

Comment:
```diff
-SOURCE_URI="https://forge.imag.fr/frs/download.php/836/ocl-icd-2.2.12.tar.gz"
+SOURCE_URI="https://forge.imag.fr/frs/download.php/836/ocl-icd-$portVersion.tar.gz"
```
Hmm, there's the "836" which will probably change, so maybe you should ignore this suggestion.